Open Judge System/Services/OJS.Services/Data/SubmissionsForProcessing/SubmissionsForProcessingDataService.cs

```diff
@@ -14,9 +20,17 @@
     public class SubmissionsForProcessingDataService : ISubmissionsForProcessingDataService
     {
         private readonly IEfGenericRepository<SubmissionForProcessing> submissionsForProcessing;
+        private readonly IEfGenericRepository<Submission> submissions;
 
-        public SubmissionsForProcessingDataService(IEfGenericRepository<SubmissionForProcessing> submissionsForProcessing) =>
+        private readonly WorkerType defaultWorkerType;
+        public SubmissionsForProcessingDataService(
+            IEfGenericRepository<SubmissionForProcessing> submissionsForProcessing,
+            IEfGenericRepository<Submission> submissions)
```

Comment:
Not able to use the dataService for submissions The configuration is invalid. The following diagnostic warnings were reported:
-[Lifestyle Mismatch] SubmissionsForProcessingDataService (Hybrid Web Request / Async Scoped) depends on SubmissionsDataService (Transient).
-[Short Circuited Dependency] SubmissionsForProcessingDataService might incorrectly depend on unregistered type SubmissionsDataService (Transient) instead of ISubmissionsDataService (Hybrid Web Request / Async Scoped).